apps/web/src/lib/file-utils.ts

```
@@ -0,0 +1,116 @@
+import type { FileItem } from "./types";
+
+/**
+ * Format file size in human readable format
+ */
+export function formatFileSize(bytes?: string | number): string {
+	if (!bytes) return "—";
+
+	const size = typeof bytes === "string" ? parseInt(bytes) : bytes;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)